src/rocstar.js

```diff
@@ -0,0 +1,223 @@
+    const balance = await shibuyaHelper.getBalance(proxyOnParachain);
+
+    if (balance.free.lt(minBalance)) {
+        console.log('\nb) Topping up the proxy account on Shibuya with SBY ...\n');
+        const amount = new BN(1000, 10);
+        const amountBN = amount.mul(decimalBN);
+        const topUpExtrinsic = shibuyaHelper.api.tx.balances.transfer(proxyOnParachain, amountBN.toString());
+        await sendExtrinsic(shibuyaHelper.api, topUpExtrinsic, keyPair);
+    } else {
+        const freeAmount = (new BN(balance.free)).div(decimalBN);
+        console.log(`\nb) Proxy’s balance is ${freeAmount.toString()}, no need to top it up with SBY transfer ...`);
+    }
+
+    console.log('\n2. One-time proxy setup on Turing');
+    console.log(`\na) Add a proxy for Alice If there is none setup on Turing (paraId:${shibuyaHelper.config.paraId})\n`);
+    const proxyTypeTuring = 'Any';
+    const proxyOnTuring = turingHelper.getProxyAccount(turingAddress, shibuyaHelper.config.paraId);
+    const proxyAccountId = keyring.decodeAddress(proxyOnTuring);
+    const proxiesOnTuring = await turingHelper.getProxies(turingAddress);
+    const proxyMatchTuring = _.find(proxiesOnTuring, { delegate: proxyOnTuring, proxyType: proxyTypeTuring });
+
+    if (proxyMatchTuring) {
+        console.log(`Proxy address ${proxyOnTuring} for paraId: ${shibuyaHelper.config.paraId} and proxyType: ${proxyTypeTuring} already exists; skipping creation ...`);
+    } else {
+        console.log(`Add a proxy of ${parachainName} (paraId:${shibuyaHelper.config.paraId}) and proxyType: ${proxyTypeTuring} on Turing ...\n Proxy address: ${proxyOnTuring}\n`);
+        await sendExtrinsic(turingHelper.api, turingHelper.api.tx.proxy.addProxy(proxyOnTuring, proxyTypeTuring, 0), keyPair);
+    }
+
+    // Reserve transfer SBY to the proxy account on Turing
+    const minBalanceOnTuring = new BN(MIN_BALANCE_IN_PROXY).mul(decimalBN);
+    const balanceOnTuring = await turingHelper.getTokenBalance(proxyOnTuring, SHIBUYA_TOKEN_ID_ON_TURING);
+
+    if (balanceOnTuring.free.lt(minBalanceOnTuring)) {
+        console.log(`\nb) Topping up the proxy account on ${turingChainName} via reserve transfer ...`);
+        const topUpAmount = new BN(1000, 10);
+        const topUpAmountBN = topUpAmount.mul(decimalBN);
+        const reserveTransferAssetsExtrinsic = shibuyaHelper.createReserveTransferAssetsExtrinsic(turingHelper.config.paraId, proxyAccountId, topUpAmountBN);
+        await sendExtrinsic(shibuyaHelper.api, reserveTransferAssetsExtrinsic, keyPair);
+    } else {
+        const freeBalanceOnTuring = (new BN(balanceOnTuring.free)).div(decimalBN);
+        console.log(`\nb) Proxy’s balance is ${freeBalanceOnTuring.toString()}, no need to top it up with reserve transfer ...`);
+    }
+
+    console.log(`\n3. Execute an XCM from ${parachainName} to schedule a task on ${turingChainName} ...`);
+
+    const result = await scheduleTask({
```

Comment:
I haven’t tested this rocstar.js code. @imstar15 could you test it? Load your wallet json to this program, and send me your wallet address for some Rocstar.